api/models.py

```diff
@@ -107,6 +109,13 @@ class Meta:
         related_name="tasks"
     )
 
+    created_by = models.ForeignKey(
+        User,
+        verbose_name='Created by',
+        related_name='task_created',
+        blank=True
```

Comment:
I'm not sure with this line. I think it may be good to have this field required.